cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -960,19 +963,19 @@
     <property>
         <name>kafka.log.dir</name>
         <value>/tmp/kafka-logs</value>
-        <description>Directory to store Kafka logs</description>
+        <description>Kafka log storage directory</description>
     </property>
 
     <property>
         <name>kafka.zookeeper.namespace</name>
         <value>kafka</value>
-        <description>ZK namespace for Kafka</description>
+        <description>Kafka ZooKeeper namespace</description>
     </property>
 
     <property>
         <name>kafka.default.replication.factor</name>
         <value>1</value>
-        <description>Kafka replication factor</description>
+        <description>Kafka replication factor [See Note 1]</description>
```

Comment:
I don't think we should have a reference from ``cdap-default.xml`` to the rst file. People may want to look at cdap-default.xml (in their source tree) by itself, and would have difficulty in finding the note. Is it necessary for this to be a reference? Can we just pull in the text from Note 1 into the description here?